tests/nodes/test_config.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
This file should probably be at a higher level, or a sibling folder, since it's not really core `nodes` code.

Reply:
Moved